.ado/azure-pipelines.publish.yml

```diff
@@ -49,6 +49,11 @@ jobs:
           yarn test
         displayName: 'yarn test [test]'
 
+      - script: |
```

Comment:
I'm curious why the step above doesn't use `yarn bundleci` which is supposed to run tests, bundles, depcheck etc....

Reply:
it would only do work for 2 other projects that have bundle too: apps\win32 and apps\fluent-tester

Reply:
I'm not sure the history behind that. @Saadnajmi It looks like you added that two years ago. I know that's a long time ago, but do you see any reason we can't use `yarn buildci` instead of `yarn test`?

According to lage.config.js - `buildci: ['build', 'test', 'depcheck', 'bundle'],` and `test: ['build-tools', 'lint', 'build'],`.

I don't think there should be any issue?

Reply:
It's a bit before my time, but:

yarn buildci was basically "Run most of the CI as one command, so that devs could run it locally to debug issues". That's effectively the "JS PR" job. There was a deliberate attempt to keep the building and bundling separate because those are time intensive tasks. 

yarn test is a subset of yarn buildci, and I guess what the platform jobs use. Each platform Job (Apple PR, Windows PR, etc) still end up bundling their platform, as a pre-step to build the test app.

For the publish pipeline, I kinda feel like we want neither. Publish IMO means we already passed PR checks so why are we testing stuff again? But it was there already and was maybe some good extra validation, so I left it there

Reply:
Did some digging - it looks like other similar GitHub repos (FI + sdx-platform) run their version of `yarn buildci`. All of them build and bundle.

And if we ever have the need to publish bundles like in this PR, it will be done automatically without the need to add a one-off script like I have above. I'm going to change `yarn test` to `yarn buildci` unless anyone has **strong** opinions otherwise.

Reply:
@rurikoaraki Do you see any issue with what I mentioned above? Any string opinion?

Reply:
Yeah that seems fine to me.